templates/opensshd.conf.j2

```diff
@@ -203,13 +203,14 @@ X11UseLocalhost yes
 
 {% if ssh_server_permit_environment_vars -%}
 PermitUserEnvironment yes
-{% for item in ssh_server_permit_environment_vars -%}
-AcceptEnv {{ item }}
-{% endfor %}
 {% else -%}
 PermitUserEnvironment no
 {% endif %}
 
+{% if ssh_server_accept_env_vars -%}
```

Comment:
Can you add this variable and `ssh_server_permit_environment_vars` to the test `tests/default_custom.yml` with some real values?